result_classes.py

```diff
@@ -0,0 +1,81 @@
+"""The data structure of all result kinds."""
```

Comment:
nit: let's not call this "result_classes". How about just "results" ?

Reply:
Yep, fixed.
Should it be in a separated dir as well (e.g., `experiment` or `common`)?

Reply:
we can just keep it where it is for now and move it later if needed.

Reply:
Shall we merge this?
The next step is enabling Prototyper for cloud experiments.
Once done, we will be able to run a full experiment.